README.md

```diff
@@ -1,4 +1,6 @@
+[![Build Status](https://travis-ci.org/zharikovpro/railgun.svg?branch=master)](https://travis-ci.org/zharikovpro/railgun)
```

Comment:
I think that for correct work this icons you need to connect the repository `zharikovpro/railgun` on the server Travis CI